ui/yarn.lock

```diff
@@ -0,0 +1,10051 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I also don't think there's a need to check this one as we are using npm to run the application.

Reply:
I believe we can switch to just using npm fully rather than yarn. This should be removed and a package.lock file created instead in my opinion.